### DISCUSSION TITLE
daisyUI for Python

### DISCUSSION THREAD

**Discussion by douglaslassance** [saadeghi/daisyui#2439] [Ideas / feature requests]
We have [pytailwindcss](https://pypi.org/project/pytailwindcss/). It would be nice to have daisyUI.

**Comment by deviprsd** [comment]
@douglaslassance Your pytailwindcss should accept plugins, daisyUI is just a plugin from tailwind. I'm not sure why there needs to be another tooling to support that. You should make it a feature request to [pytailwindcss](https://pypi.org/project/pytailwindcss/) team

> It's not all roses, though. Giving up Node.js means you won't be able to install plugins or additional dependencies for your Tailwind CSS setup. At the same time, that might not be a dealbreaker. You can still customize Tailwind CSS via the tailwind.config.js file. And the standalone build also comes with all official Tailwind CSS plugins like @tailwindcss/aspect-ratio, @tailwindcss/forms, @tailwindcss/line-clamp, and @tailwindcss/typography. So in 90% of Tailwind CSS usage cases you should be covered, and the setup is so simplified now.

**Reply by douglaslassance** [reply]
Makes sense. I'll be testing the `pytailwindcss` plugins feature.
